### ISSUE TITLE
Request: Sound for Pomodoro start (end of break)

### ISSUE THREAD

**Issue by t0mm1** [drakegawain/PomoBot#9]
I'll keep it short: can we have a sound for the end of a break as well to know when to get back to the next pomodoro? Thanks for your work!

**Comment by drakegawain** [comment]
Yes! I'm going to work on this soon! Thanks for using this bot! Any suggestions for improvement, please feel free to say!
